src/views/collection-view/components/collection-meta/collection-meta.module.css

```diff
@@ -0,0 +1,45 @@
+.icon {
+  color: var(--token-color-foreground-primary);
+}
+
+.heading {
+  margin-top: 12px;
+}
+
+.description {
+  margin-top: 8px;
+  color: var(--token-color-foreground-primary);
+}
+
+.cta {
+  display: flex;
+  align-items: center;
+  margin-top: 24px;
+  margin-bottom: 16px;
+
+  @media (--dev-dot-tablet-up) {
+    margin-top: 32px;
+    margin-bottom: 24px;
+  }
+}
+
+.ctaButton {
+  composes: root from '/src/components/button/button.module.css';
+  composes: primary from '/src/components/button/button.module.css';
+  composes: medium from '/src/components/button/button.module.css';
```

Comment:
Decided to just composes button styles from that component. Lmk if there's a better way to do this.

Reply:
Thinking a little farther out... maybe we should sync with design on this? Eg do we need to solve for the same "things that look like `Button` and have all the same variations, but could be a `<a />` or a `<button />`" use case that `@hashicorp/react-button` solves for? 🤔 

Or maybe they'd be willing to switch this up to a more standard `StandaloneLink` in order to avoid this whole conundrum?

Reply:
@zchsh okay I made a new component and copied over the styles from 'Button'. There's still some more work to do on that component to make it fully legit, so I added some comments. But I figure for the sake of this PR, that fine-grained work isn't necessary. Take a look and lmk what you think!